### ISSUE TITLE
Error in React project when formatOnSaveMode not set to "file"  

### ISSUE THREAD

**Issue by mattbrannon** [idahogurl/vs-code-prettier-eslint#46]
**Describe the bug**
When using the extension in a React project, the extension will error when `formatOnSaveMode` is set to `modifications` or `modificationsIfAvailable`.

**To Reproduce**
Steps to reproduce outlined in [example repo](https://github.com/mattbrannon/prettier-eslint-format-on-save-mode-react)

**Expected behavior**
The extension should not error

**Screenshots**

**Versions (please complete the following information):**
- VS Code Prettier ESLint extension `5.0.3`
- Visual Studio Code: `1.68.1`
- Node: `16.15.1`
- Package Manager `yarn 1.22.19`
- prettier: `2.7.1`
- eslint: `8.19.0`

**System Specifications (please complete the following information):**
- OS: `Mac OS Mojave 10.14.6`
- Processor: `2.2 GHz Intel Core i7`
- RAM Size: `8 GB`
 

The problem seems to be specific to React as I was unable to reproduce in a vanilla js project. A potential solution (at least temporarily) would be to make sure `formatOnSaveMode` is always set to `file`. 

I have my workspace `.vscode/settings.json` set like this and it works great. 
```json
{
  "editor.defaultFormatter": "rvest.vs-code-prettier-eslint",
  "editor.formatOnPaste": false,
  "editor.formatOnType": false,
  "editor.formatOnSave": true,
  "editor.formatOnSaveMode": "file",
  "files.autoSave": "onFocusChange"
}
```

**Comment by idahogurl** [comment]
@mattbrannon  I was able to reproduce the behavior by wrapping the `<div></div>` in `App.js` with an empty fragment and then saving.

**Before Modification**
```<div></div>```

**After Modification**
```<><div></div></>``` 

VS Code will send ``<>`` and then `</>` to my format function. Prettier is giving an error when trying to format this since it uses Babel. Babel won't successfully parse `<>` alone. I can update the documentation but since "file" is the default for "formatOnSaveMode", I'm going to close this issue.

**Comment by mattbrannon** [comment]
I think updating the documentation is enough. I was banging my head against the wall for a while and then once I changed the setting back to "file" it worked like a charm. I think having some explicit instructions letting people know that it will break if `formatOnSaveMode` is set to anything but `"file"` would be helpful to others who may find themselves in a similar situation. 

In general, the docs have all the information someone needs to get the extension working. But there's a few places that I think cause a little friction for users like myself when first attempting to install and use the it. If you're open to it I could submit a pull request with some ideas.

**Comment by idahogurl** [comment]
@mattbrannon I'd love a PR. I'll reopen this issue so you can close it with your PR.

**Comment by idahogurl** [comment]
Closed by #49
